model/model.js

```diff
@@ -0,0 +1,6 @@
+const runners = require('../json_files/runners.json');
+
+
+module.exports = {
+    runners
+} 
```

Comment:
The filename of this file (`model.js`) would probably be better as being the name of what it's modelling. Say you had more models (e.g. matches, locations, seasons etc...) then you couldn't call all the files `model.js`. I would call it something like `runners.js`)